.gitignore

```diff
@@ -6,3 +6,5 @@
 /laminas-mkdoc-theme/
 /phpunit.xml
 /vendor/
+/documentation-theme/
+.markdownlint.json
```

Comment:
Why are these entries needed?

Reply:
These are for local development - If you're checking the docs locally, it'll prevent you from accidentally committing those files. Want a revert?

Reply:
I have no problem with the entries, I asked because we did not have them before.